tests/sentry/issues/endpoints/test_organization_group_index.py

```diff
@@ -1903,14 +1903,6 @@ def test_expand_owners(self):
         )
         assert response.data[0]["owners"][2]["type"] == GROUP_OWNER_TYPE[GroupOwnerType.CODEOWNERS]
 
-    @override_settings(SENTRY_SELF_HOSTED=False)
```

Comment:
This is not a useful test, we already test the rate limiting capabilities directly